treemap/index.js

```diff
@@ -0,0 +1,152 @@
+    // .width(346.43)// 6 col
+    // .width(74)// markets std print
+        .height(200), // markets std print
+
+    social: gChartframe.socialFrame(sharedConfig)
+        .margin({
+            top: 140, left: 50, bottom: 138, right: 40,
+        })
+    // .title("Put headline here")
+        .width(612)
+        .height(612), // 700 is ideal height for Instagram
+
+    video: gChartframe.videoFrame(sharedConfig)
+        .margin({
+            left: 207, right: 207, bottom: 210, top: 233,
+        }),
+    // .title("Put headline here")
+};
+
+
+// add the frames to the page...
+d3.selectAll('.framed')
+    .each(function addFrames() {
+        const figure = d3.select(this);
+        figure.select('svg')
+            .call(frame[figure.node().dataset.frame]);
+    });
+parseData.load(dataFile)
+    .then(({ seriesNames, plotData }) => {
+        Object.keys(frame).forEach((frameName) => {
+            const currentFrame = frame[frameName];
+
+            // define other functions to be called
+            const myLegend = gLegend.legend();// sets up the legend
+            // const plotDim=currentFrame.dimension()//useful variable to carry the current frame dimensions
+            // const tickSize = currentFrame.dimension().width;// Used when drawing the yAxis ticks
+            const myTreemap = treemap.draw();
+
+            // create a 'g' element at the back of the chart to add time period
+            // highlights after axis have been created
+            const axisHighlight = currentFrame.plot().append('g');
+
+            axisHighlight.append('rect')
+                .attr('width', currentFrame.dimension().width)
+                .attr('height', currentFrame.dimension().height)
+                .attr('fill', '#ededee');
+
+            myTreemap
+                .plotDim(currentFrame.dimension())
+                .seriesNames(seriesNames)
+                .rem(currentFrame.rem())
+                .colourPalette((frameName));
+
+            currentFrame.plot()
+                .selectAll('.treemap')
+                .data(plotData)
+                .enter()
+                .append('g')
+                .attr('class', 'treemap')
+                .call(myTreemap);
+
+            myLegend
+                .seriesNames(seriesNames)
+                .geometry(legendType)
+                .frameName(frameName)
+                .rem(currentFrame.rem())
+                .alignment(legendAlign)
+                .colourPalette((frameName));
+
+            currentFrame.plot()
+                .append('g')
+                .attr('id', 'legend')
+                .selectAll('.legend')
+                .data(seriesNames)
+                .enter()
+                .append('g')
+                .classed('legend', true)
+                .call(myLegend);
+
+            const legendSelection = currentFrame.plot().select('#legend');
+        const legheight = (legendSelection.node().getBBox().height); // eslint-disable-line
```

Comment:
I didn't catch this line in linting because it disables the linter, I'm not sure why? It's misaligned at any rate, and you don't need the parentheses.

Reply:
Its not misaligned as such, but this just moves it above the plot area and sits it nicely